eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxInfo.hs

```diff
@@ -444,28 +444,28 @@ instance FromCBOR FailureDescription where
       dec 1 = SumD PlutusFailure <! From <! From
       dec n = Invalid n
 
-data ScriptResult = Passes | Fails ![FailureDescription]
+data ScriptResult = Passes ![PlutusDebug] | Fails ![FailureDescription]
   deriving (Show, Generic, NoThunks)
 
 instance ToCBOR ScriptResult where
-  toCBOR Passes = encode $ Sum Passes 0
+  toCBOR (Passes ps) = encode $ Sum Passes 0 !> To ps
   toCBOR (Fails fs) = encode $ Sum Fails 1 !> To fs
 
 instance FromCBOR ScriptResult where
   fromCBOR = decode (Summands "ScriptResult" dec)
     where
-      dec 0 = SumD Passes
+      dec 0 = SumD Passes <! From
```

Comment:
Note that the `Passes` constructor (unlike `Fails` below) is not a part of any predicate failure. Therefore this change to the serialization does _not_ break the node-to-client protocol.